frontend/src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { DataGrid, GridActionsCellItem } from '@mui/x-data-grid';
+import { Button } from '@mui/material';
+
+const columns = [
+  { field: 'id', headerName: 'ID', width: 70 },
+  { field: 'scholarshipName', headerName: 'Scholarship Name', width: 130 },
+  { field: 'startDate', headerName: 'Start Date', type: "date", width: 130 },
+  { field: 'endDate', headerName: 'End Date', type: "date", width: 130 },
+  { field: 'status', headerName: 'Status', width: 130 },
+  { field: 'actions', headerName: 'Actions', type: "actions", width: 130,
```

Comment:
Let's make the width bigger then adjust the width of the whole table

Reply:
Thank you, I have adjusted this part 🙏